### DISCUSSION TITLE
frpntend: 44.203.187.172    not able to run shipping

### DISCUSSION THREAD

**Discussion by Nazrina412** [learndevopsonline/learndevopsonline#1933] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/3e7fc8ce-6b28-46e8-beba-2d1d21da021f)

**Comment by chaithanya1812** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/895a2bb8-491b-4b0d-b4d0-b69e0f17fcf6)
